cmd/sandbox/start.go

```diff
@@ -68,9 +70,14 @@ func startSandboxCluster(ctx context.Context, args []string, cmdCtx cmdCore.Comm
 
 	reader, err := startSandbox(ctx, cli, os.Stdin)
 	if err != nil {
-		return err
+		if err.Error() != clierrors.ErrSandboxExists {
+			return err
```

Comment:
With your previous change exit code was 0. This will cause a non-zero exit code . Is that what was desired in the first place

Reply:
!= . got it

Reply:
yes